dvc/scm/git/backend/dulwich.py

```diff
@@ -102,8 +169,12 @@ def list_tags(self) -> Iterable[str]:
     def list_all_commits(self) -> Iterable[str]:
         raise NotImplementedError
 
-    def get_tree(self, rev: str, **kwargs) -> BaseTree:
-        raise NotImplementedError
+    def get_trie(self, rev: str, **kwargs) -> DulwichTrie:
+        from dulwich.objectspec import parse_tree
+
+        rev = self.scm.resolve_rev(rev)
```

Comment:
Using `self.scm` is a bad thing (we do it in stash code too). Caused by a lack of implementation for `dulwich`. Will be solved in the future.